src/Model/Contact.php

```diff
@@ -1197,8 +1199,20 @@ public static function photoMenu(array $contact, int $uid): array
 			$pm_url = 'message/new/' . $contact['id'];
 		}
 
+		if (in_array($contact['rel'], [contact::SHARING])) {
```

Comment:
Same question about targeting groups, this would match all the contacts you are following.

Reply:
```suggestion
		if ($contact['contact-type'] == self::TYPE_COMMUNITY) {
```

Reply:
changed the code to your suggestion. it works.